test/Microsoft.TemplateEngine.Cli.UnitTests/TemplatePackageCoordinatorTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using FakeItEasy;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Abstractions.Installer;
+using Microsoft.TemplateEngine.Abstractions.TemplatePackage;
+using Microsoft.TemplateEngine.Cli.Commands;
+using Microsoft.TemplateEngine.Edge.Settings;
+using Microsoft.TemplateEngine.Utils;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.Cli.UnitTests
+{
+    public class TemplatePackageCoordinatorTests : IClassFixture<TemplatePackageCoordinatorTests.ConsoleRedirector>
+    {
+        public class ConsoleRedirector
```

Comment:
Small concern if this can impact other tests that might be doing console output for other purposes.